### PR TITLE
List Structure

### PR DESCRIPTION
**Pull request by melaniesigrid**
## Hello there! :alien:

Thank you for reviewing my project. On this PR:
- I created the HTML main structure for my list
-  Added CSS style to match the one given
- Created an array of objects that will be each task on the list.
- Created a function that iterates through the objects and adds them as HTML elements.

Your comments and suggestions to make the project better will be greatly appreciated! :star:

### CODE REVIEW

File: webpack-demo/webpack.config.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ module.exports = {
   },
   plugins: [
     new HtmlWebpackPlugin({
-      template: './src/index.html'
+      template: './src/index.html',
```
Comment (GzimAsani) [inline webpack-demo/webpack.config.js:12]: - [x] Good job with  your javascript code :+1: . But  on page load, the task array should be rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @melaniesigrid                                    
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Great job with the project it looks really good :muscle: 

## Required Changes ♻️ 
- [x] Check the comments below :arrow_double_down: 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this 
You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: webpack-demo/webpack.config.js (hunk 1 above)
Comment (melaniesigrid) [reply webpack-demo/webpack.config.js:12]: Thank you very much for the code review. I took care of the issues.

**Review by HENRYKC24 — APPROVED** [review]
Approved
### Hi @melaniesigrid :wave:
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**